### PR TITLE
set promptbox opacity

### PR DESCRIPTION
**Pull request by Goober5000**
Set gr.CurrentOpacityType before drawing the prompt box so it isn't affected by fade-ins, etc.

### CODE REVIEW

**Review by BMagnu — COMMENTED** [review]
Should this be a config?
I could imagine people wanting both

**Comment by Goober5000** [comment]
I don't think so.  If the prompt box is translucent, it's difficult to make a choice.  Here is an example from Series Resurrecta, which is where I originally spotted the issue:

prompt

File: PromptBox/data/tables/proBox-sct.tbm [hunk 1]
```diff
@@ -463,6 +463,8 @@ end
 
 function PromptBox:Draw()
 
+	gr.CurrentOpacityType = GR_ALPHABLEND_NONE
```
Comment (EatThePath) [inline PromptBox/data/tables/proBox-sct.tbm:467]: for safety's sake I'd ask that you store what the opacity type was before this and reset it back to that after drawing. You never know when some weird thing will make assumptions about this shared state...

**Review by EatThePath — CHANGES_REQUESTED** [review]

File: PromptBox/data/tables/proBox-sct.tbm (hunk 1 above)
Comment (Goober5000) [reply PromptBox/data/tables/proBox-sct.tbm:467]: Fair point.  Updated.

**Review by EatThePath — APPROVED** [review]

**Pull request merged**